documentation/docs/get-started/did-web/how-to-create/hosting-did-web.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_label: 'Using Your Own Domain or Ngrok'
+sidebar_position: 2
+---
+
+# Create and Host a did:web Identifier
+
+## Section 1: Domain Setup and Importance of HTTPS
+
+### Why HTTPS is Essential
+
+For `did:web` identifiers, it's crucial that your domain is HTTPS-enabled. The DID method `did:web` relies on web infrastructure to serve the DID Document from a domain. Without HTTPS, your DID Document won't be considered secure or verifiable by external services and clients.
+
+Most browsers and services enforce HTTPS to ensure that data exchanged between servers and users remains encrypted and protected from interception.
+
+### Preparing Your Domain
+
+To create a `did:web`, you'll need:
+
+- A registered domain (e.g., `example.com`).
+- The ability to host files on that domain.
+- HTTPS enabled on your domain, typically via SSL certificates (e.g., via Let's Encrypt or other providers).
+
+Once your domain is set up with HTTPS, you can proceed to create and host your `did:web` identifier.
+
+#### Alternative: Testing with Ngrok (Without a Domain)
+
+If you don't have a domain but want to test the process locally, you can use **ngrok** to create a secure HTTPS tunnel to your localhost. Ngrok temporarily exposes your local server to the internet, giving you a unique URL for testing.
+
+**Steps to Use Ngrok:**
+
+1. Install and register **ngrok**.
```

Comment:
I think the word **ngrok** could be turned into a hyperlink, so users can click on it and be redirected to the https://ngrok.com/download site.